harvard-lts/jstor_aspace_transformer
# Ss 114 subtask ss 74

**The title of this pull-request should be a brief description of what the pull-request fixes/improves/changes. Ideally 50 characters or less.**
* * *

**JIRA Ticket**: [SS-74](https://jira.huit.harvard.edu/browse/SS-74)

* [Other Relevant Links (Mailing list discussion, related pull requests, etc.)
]*
(https://jira.huit.harvard.edu/browse/SS-58)
(https://jira.huit.harvard.edu/browse/SS-218)

# What does this Pull Request do?
XSLT changes for retrospective/ongoing jstor processing to via; test documents for all; unit tests for all

# How should this be tested?

Check out this branch
Start up docker
exec into docker and run > pytest
itests should still work

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? Y
- integration tests?

# Interested parties
Tag (@ mention) interested parties
